cg/cli/workflow/nextflow/options.py

```diff
@@ -47,3 +47,16 @@
 OPTION_OUTDIR = click.option(
     "--outdir", type=click.Path(), help="Path to output folder containing analysis files"
 )
+
+
+OPTION_CONFIG = click.option(
+    "-config",
```

Comment:
Maybe --config instead? Seems like we mostly use `--` for "long" option names

Reply:
That is a nextflow standard. I think it's best if we keep it to avoid confusion. Nextflow uses `-` for their general options and `--` for pipeline specific ones.

Reply:
Ok, got it!